.nvmrc

```diff
@@ -0,0 +1 @@
+v8.14.1
```

Comment:
The reason why I do this is because of the `package-lock.json` file. Different node/npm versions cause different outputs. At least this is what I experienced in the past. Haven't verified this lately again.

This can limit the diff in future PRs when someone uses a different node/npm version without really changing the lock file. Don't you think this could be helpful?